app/app/src/components/AppForm.svelte

```diff
@@ -95,6 +95,7 @@ async function onChange(event: Event)
 				<input
 					type="file"
 					class="form-input"
+					accept="message/rfc822"
```

Comment:
Add filename extension in addition to MIME type string for better support.

```suggestion
					accept=".eml,message/rfc822"
```

Reply:
Indeed it should be enough. Sounds fine!
I hope we can trust browsers to consistently recognize eml files like this then.